test/artefacts/overseas-passports/falkland-islands/replacing/adult.txt

```diff
@@ -17,7 +17,9 @@ Don’t book travel until you have a valid passport. If you need to travel more
 
 ## Cost
 
-You’ll have to pay a fee for your passport and a courier fee of £9.70. The courier fee pays for your passport to be sent back to you securely.
+You’ll have to pay a fee for your passport and a courier fee of £9.70.
+
+The courier fee pays for your passport to be sent back to you securely.
```

Comment:
I'm afraid we (@chrisroos and I) also don't know what "IPS1" countries are. We're happy to accept the changes to the regression test artefacts as they match the change we've made to the template. We've added the following message to each of the commit notes that mention IPS1/2/3:

> We don't know what defines IPS1 countries. This just seemed like a
natural way to split the changes in the branch. We're happy to accept
the changes to the regression test artefacts because they match the
change we made to the template.